### PR TITLE
fix: `conf` defaults to `{}`

### PR DESCRIPTION
**Pull request by Alex-Joseph**
Description:

version: `4.1.0`

I was getting the error `TypeError: Cannot read property 'runOnly' of undefined` from this line:

`axe.default(React, ReactDom, 1000);`

The solution was similar to [this author's solution](https://github.com/dequelabs/axe-core-npm/issues/157#issuecomment-758724273), therefore perhaps they are related?

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [x] Code is reviewed for security

### CODE REVIEW

File: packages/react/index.ts [hunk 1]
```diff
@@ -347,23 +348,23 @@ function reactAxe(
...
+  const runOnly = conf['runOnly'];
   if (runOnly) {
     conf['rules'] = axeCore
       .getRules(runOnly)
       .map(rule => ({ ...rule, id: rule.ruleId, enabled: true }));
     conf['disableOtherRules'] = true;
   }
 
-  if (conf) {
+  if (!isEmpty(conf)) {
```
Comment (stephenmathieson) [inline packages/react/index.ts:367]: I don't this this justifies bringing in a dependency. Can we do this without `ramda`?

Comment (Alex-Joseph) [inline packages/react/index.ts:367]: Hmm that is a fair point. What about this?

```suggestion
  if (Object.keys(conf).length > 0) {
```

File: packages/react/index.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import axeCore = require('axe-core');
 import rIC = require('requestidlecallback');
 import after = require('./after');
+import { isEmpty } from 'ramda';
```
Comment (Alex-Joseph) [inline packages/react/index.ts:5]:
```suggestion

```

**Review by Alex-Joseph — COMMENTED** [review]
Suggested change

File: packages/react/index.ts (hunk 1 above)
Comment (stephenmathieson) [reply packages/react/index.ts:367]: Looks good to me!

**Comment by n370** [comment]
👍

**Review by jeeyyy — APPROVED** [review]

**Comment by stephenmathieson** [comment]
Thanks @Alex-Joseph!

**Pull request merged**